common/src/main/java/com/wynntils/utils/render/FontRenderer.java

```diff
@@ -397,18 +394,17 @@ public float calculateRenderHeight(List<TextRenderTask> toRender) {
     }
 
     public float calculateRenderHeight(List<StyledText> lines, float maxWidth) {
-        int sum = 0;
-        for (StyledText line : lines) {
-            sum += font.wordWrapHeight(line.getString(), (int) maxWidth);
-        }
-        return sum;
+        return lines.stream()
+                .mapToInt(line -> (int) calculateRenderHeight(line, maxWidth))
+                .sum();
```

Comment:
This int cast was stupid from me, fixed in https://github.com/Wynntils/Artemis/pull/2358/commits/2316d197cbbedbe2fef72c10a8371fe139a07b63.